pages/settings/teams.tsx

```diff
@@ -1,46 +1,59 @@
+import { GetServerSideProps } from "next"
 import Head from 'next/head';
-import prisma from '../../lib/prisma';
-import Modal from '../../components/Modal';
 import Shell from '../../components/Shell';
 import SettingsShell from '../../components/Settings';
 import { useEffect, useState } from 'react';
+import type { Session } from "next-auth"
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689300)
```suggestion
import type { Session } from "next-auth";
```